File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-a-chart-in-a-worksheet-using-radio-buttons_3.cs
        private void barChart_CheckedChanged(object sender, EventArgs e)
        {
            if (((RadioButton)sender).Checked)
            {
                this.selectedType = Microsoft.Office.Interop.Excel.XlChartType.xlBarClustered;
                if (this.SelectionChanged != null)
                {
                    this.SelectionChanged(this, EventArgs.Empty);
                }
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-a-chart-in-a-worksheet-using-radio-buttons_3.vb
    Private Sub barChart_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) _
        Handles barChart.CheckedChanged

        If (CType(sender, RadioButton).Checked) Then

            Me.selectedType = Microsoft.Office.Interop.Excel.XlChartType.xlBarClustered
            RaiseEvent SelectionChanged(Me, EventArgs.Empty)
        End If
    End Sub